cli/changelog/0.18.0-rc.5.md

```diff
@@ -0,0 +1,5 @@
+### Features
+
+- Add initial support for custom resolvers written in JavaScript or TypeScript.
+- Add `unlink` and `link` commands.
+- Change the auth configuration to require an API key in the locally running gateway.
```

Comment:
Not important, but could mention "Fixed some inefficiencies in handling introspection queries" or similar

Reply:
Of course!

Reply:
I have only gone through the CLI repo history. Now I wonder what else is worth mentioning from the gateway work. Maybe query planning? But I guess better to include that when @Miaxos enables it